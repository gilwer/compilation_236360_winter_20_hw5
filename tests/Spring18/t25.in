int doSomething(byte x, bool c) {
	if (c) {
		int i = 0;
		int switch = 1;
		while (i < x) {
			switch = switch * x;
			i = i + 1;
			if (switch > 1000)
				return switch;
		}
		return switch * 10;
	}
	else {
		return x - 50;
	}
}

void main() {
	print("5,true: ");
	printi(doSomething(5b,true));
	print("100,true: ");
	printi(doSomething(100b,true));
	print("5,false: ");
	printi(doSomething(5b,false));
	print("100,false: ");
	printi(doSomething(100b,false));
	//print("");
}