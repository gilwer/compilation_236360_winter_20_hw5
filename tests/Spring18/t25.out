5,true: 
3125
100,true: 
10000
5,false: 
-45
100,false: 
50
